config/settings.yml

```diff
@@ -871,17 +856,8 @@
   :level_rails: info
   :level_api: info
   :level_automation: info
-  :level_azure_stack: info
-  :level_cn_monitoring: info
   :level_fog: info
-  :level_gce: info
-  :level_mw: info
```

Comment:
Note that level_mw and level_cn_monitoring are being dropped here.  I might need a data migration to drop these from SettingsChanges.